backend/dataall/core/environment/models.py

```diff
@@ -0,0 +1,33 @@
+"""The package contains the database models that are related to the environment"""
+
+from sqlalchemy import Column, String, ForeignKey
+from dataall.db import Resource, Base
+
+
+class EnvironmentParameter(Base):
+    """Represent the parameter of the environment"""
+    __tablename__ = 'environment_parameters'
+    environmentUri = Column(String, ForeignKey("environment.environmentUri"), primary_key=True)
+    key = Column('paramKey', String, primary_key=True)
+    value = Column('paramValue', String, nullable=True)
+
```

Comment:
I am not understanding the primary keys here. `key` is not going to be unique and environmentUri will be duplicated. Can we have a unique primary key=environmentUri-key?

Reply:
It's already the same way as you described. It confused me a little bit too at the beginning, but it's the way how SQLAlchemy creates composite keys.